pkg/testcommon/testcommon.go

```diff
@@ -241,3 +241,12 @@ func ContainerCheck(checkName string, checkState string) (bool, error) {
 
 	return false, errors.New("unable to find container " + checkName)
 }
+
+// TimeTrack determines the amount of time a function takes to return. Timing starts when it is called.
+// It returns an anonymous function that, when called, will print the elapsed run time.
+func TimeTrack(start time.Time, name string) func() {
```

Comment:
I have to confess that I don't understand yet how the idea of returning a function for later execution is better than initializing a start value and using it as a param to a regular static function :) But it does no harm.